content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,227 @@
+==========================================
+Set up Sendcloud shipping services in Odoo
+==========================================
+
+Sendcloud is a shipping service aggregator. Sendcloud facilitates the integration of European
+shipping carriers with Odoo. Once integrated, the user can select the shipping carrier on inventory
+operations in their Odoo database.
+
+.. note::
+   Here is `Sendcloud's documentation <https://support.sendcloud.com/hc/en-us/articles
+   /360059470491-Odoo-integration>`_ on setting up the integration.
+
+.. important::
+   Odoo integration of Sendcloud does **not** work on free plans of Sendcloud.
```

Comment:
I'd actually place this `important` tag right after line 28 ending in `activate (or deactivate) the shipping carriers that will be used in the Odoo database.` Seems to be a better fit with the surrounding context that discusses account creation, no?

e.g.
```
If the user does not have a Sendcloud account, fill in the contact information and payment details.
Sendcloud will ask for a :abbr:`VAT (Value-Added Tax Identification)` number or :abbr:`EORI
(Economic Operators' Registration and Identification)` number. After completing the account setup,
activate (or deactivate) the shipping carriers that will be used in the Odoo database.

.. important::
   Odoo integration of Sendcloud does **not** work on free plans of Sendcloud.
```